src/main/java/nl/tudelft/dnainator/Main.java

```diff
@@ -1,7 +1,8 @@
 package nl.tudelft.dnainator;
 
-import javafx.application.Application;
+import com.sun.javafx.application.LauncherImpl;
```

Comment:
This is supposed to be private, internal API. Are you sure you want to rely on that?